src/main/target/DALRCF405/target.c

```diff
@@ -37,8 +37,10 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM8, CH3, PC8,  TIM_USE_MOTOR, 0, 0),   // S5 (2,4)  (2.2)
     DEF_TIM(TIM3, CH4, PB1,  TIM_USE_MOTOR, 0, 0),   // S6 (1,2)
 
+#if !defined(VIVAF4AIO)
```

Comment:
Shouldn't this be?
```
#if defined(VIVAF4AIO)
```

Reply:
@etracer65 It is use when wants to define only on base target, for avoid conflicts

Reply:
Ah, I was mistaken. I thought only the VIVAF4AIO had motors 7/8 defined and not the base target.